views/layouts/main.handlebars

```diff
@@ -0,0 +1,277 @@
+<!DOCTYPE html>
+<html class="has-navbar-fixed-top">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Oracle of Changes</title>
+
+		<!-- Font Awesome Icons from CDN -->
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
I highly dont recommend on using two different css framework for one project. Please rectify this before meet and greet